Python/Product/TestAdapter.Executor/Services/ExecutorService.cs

```diff
@@ -84,7 +84,7 @@ sealed class TestRedirector : Redirector {
 
         }
 
-        public string[] GetArguments(IEnumerable<TestCase> tests, string outputfile, string coveragePath) {
+        public string[] GetArguments(IEnumerable<TestCase> tests, string outputfile, string coveragePath, PythonProjectSettings settings) {
```

Comment:
You already have access to `_projectSettings` so no need to add it as a parameter to this method?

Reply:
since we are removing pytest from this class I thought it best to pass it in